justfile

```
@@ -28,6 +28,9 @@ test *args="//...":
 test-asan *args="//...":
   just test {{args}} --config=asan
 
+test-stream args:
```

Maybe lets call it `stream-tests` and add a shortcut for `st` so we can execute it using `just st` 
```suggestion
stream-tests args:
```

---

Nice!